forbidigo/forbidigo.go

```diff
@@ -373,10 +373,11 @@ func typeNameWithPackage(t types.Type) (typeName, packagePath string, ok bool) {
 	case *types.Named:
 		obj := t.Obj()
 		pkg := obj.Pkg()
-		if pkg == nil {
-			return "", "", false
+		pkgPath := ""
+		if pkg != nil {
+			pkgPath = pkg.Path()
 		}
-		return pkg.Name() + "." + obj.Name(), pkg.Path(), true
+		return pkgPath + "." + obj.Name(), pkgPath, true
```

Comment:
It looks like we're changing this from `pkg.Name()` to `pkg.Path()` here.  Is that safe?  The prior lack of a unit test for this method makes a it bit hard to know the intent here.

Reply:
My guess would that this is the cause of the CI failures at https://app.circleci.com/pipelines/github/ashanbrown/forbidigo/68/workflows/0124d59d-19ea-47b5-bf4f-08c408fb2962/jobs/61.  If we do want to change this, adding a unit test for this method would probably be useful as well.

Reply:
hmm. right. I'll take a look at the tests.

Reply:
All tests except
"it finds forbidden, renamed identifiers"
"import renames detected with type information"
TestLiteralAnalyzer
TestExpandAnalyzer
are good locally.
These ones panic for me on master too.

Actually changed code is not called in any of the old tests (maybe it's called in panicing tests)
tests panic on package load.

Can you run them locally?